shuup/core/models/_product_shops.py

```diff
@@ -278,22 +278,21 @@ def get_orderability_errors(  # noqa (C901)
             )
 
         if self.product.mode == ProductMode.SIMPLE_VARIATION_PARENT:
-            sellable = 0
+            sellable = False
             for child_product in self.product.variation_children.all():
                 if child_product.get_shop_instance(self.shop).is_orderable(
                         supplier=supplier,
                         customer=customer,
                         quantity=1,
```

Comment:
While at it can you fix this? I think this should be `child_shop_product.minimum_purchase_quantity` instead of 1.